pandas/tests/arrays/boolean/test_arithmetic.py

```diff
@@ -46,7 +46,7 @@ def test_add_mul(left_array, right_array, opname, exp):
 
 
 def test_sub(left_array, right_array):
-    with pytest.raises(TypeError):
+    with tm.external_error_raised(TypeError):
```

Comment:
hmm i actually think showing these errors would be good. yes they might be external, but these are important to see.

Reply:
ok done